client/src/components/TextCheckboxGroupField/TextCheckboxGroupField.js

```diff
@@ -8,10 +8,13 @@ const TextCheckboxGroupField = (props) => {
   // Map out the children and clone to set the "noHolder" prop on them.
   const childrenWithProps = React.Children.toArray(
     React.Children.map(children, (child, index) => {
-      const additionalProps = { noHolder: true };
```

Comment:
Removing noHolder is required to get validation messages showing on the TextField of this composite field